docs/src/search.md

```diff
@@ -30,55 +30,82 @@ the vector column and compute the distance.
 
 === "Python"
 
+<!--python 
+import lancedb
+import numpy as np
+uri = "data/sample-lancedb"
+db = lancedb.connect(uri)
+
+data = [{"vector": row, "item": f"item {i}"}
+     for i, row in enumerate(np.random.random((10_000, 1536)).astype('float32'))]
+
+db.create_table("my_vectors", data=data)
+-->
+
```

Comment:
These comments seem to break the tabs. What if we instead brought the setup code up above the tabs?

Also, could we just make the setup blocks normal code blocks that are inside a comment? I think that will make it much clearer to future developers what is going on. Plus IIUC it should reduce the number of steps in the code process. What do you think?

```python

<!-- Setup Code

```python
import lancedb
import numpy as np
uri = "data/sample-lancedb"
db = lancedb.connect(uri)

data = [{"vector": row, "item": f"item {i}"}
     for i, row in enumerate(np.random.random((10_000, 1536)).astype('float32'))]

db.create_table("my_vectors", data=data)
`` ` (<- Hard to escape this in GitHub)
-->

=== "Python"
```

Reply:
This would definitely make it much better and clenaer! I guess the one issue is that we would need to remove the duplicate imports in the actual code blocks, since then all of the code would go into the same file.

Reply:
Never mind, I'm completely wrong. It works with duplicate imports, I pushed the changes.